src/js/media-playback-rate-button.js

```diff
@@ -11,16 +11,34 @@ import { nouns } from './labels/labels.js';
 const DEFAULT_RATES = [1, 1.25, 1.5, 1.75, 2];
 const DEFAULT_RATE = 1;
 
+const slotTemplate = document.createElement('template');
```

Comment:
Since we removed some of the font-centric "baked in" stuff from media-chrome-button, this is the current proposed solution for buttons that will use text. (We could also add a "text-button" superclass that media-playback-rate-button extends)